src/Tizen.NUI/src/internal/EXaml/Action/CallAddMethodAction.cs

```diff
@@ -64,10 +64,10 @@ public void Init()
 
         public void OnActive()
         {
-            if (null != childOp)
+            if (null != childOp && childOp.ValueList.Count > 2 && (childOp.ValueList[0] is Instance instance) && (childOp.ValueList[1] is Instance instance1))
```

Comment:
`instance1` naming doesn't look good.
isn't it like this?
```suggestion
            if (null != childOp && childOp.ValueList.Count > 2 && (childOp.ValueList[0] is Instance instance) && (childOp.ValueList[1] is Instance child))
```

Reply:
done